app/utils/metadata.py

```diff
@@ -333,7 +333,11 @@ def get_configured_community_rules_db(
                 encoding="utf-8",
             ) as output:
                 json.dump(DEFAULT_USER_RULES, output, indent=4)
-            self.external_user_rules = DEFAULT_USER_RULES["rules"]
+            self.external_user_rules = (
+                DEFAULT_USER_RULES["rules"]
+                if isinstance(DEFAULT_USER_RULES["rules"], dict)
+                else {}
+            )
```

Comment:
So default user rules as a constant has two types I believe. But because of how dictionaries work, MyPy doesn't know statically which of the two a given key is. Even though we know that for the key rules it will always be a dict, MyPy doesn't know this. This is to just satisfy MyPy, since it thinks that if the value wasn't a dict, then you can't use .get at all.